internal/applause/applause.go

```diff
@@ -1 +1,278 @@
 package applause
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/OpenSlides/openslides-autoupdate-service/pkg/datastore"
+	"github.com/OpenSlides/openslides-icc-service/internal/iccerror"
+	"github.com/ostcar/topic"
+)
+
+const (
+	applauseInterval = time.Second
+	countTime        = 5 * time.Second
+	pruneTime        = 10 * time.Minute
+)
+
+// Backend stores the applause messages.
+type Backend interface {
+	// ApplausePublish adds the applause from a user to a meeting.
+	//
+	// The function can be called many times. The implementation of the
+	// interface has to make sure, that the applause is only counted once.
+	ApplausePublish(meetingID, userID int, time int64) error
+
+	// ApplauseSince returns the number of applause for each meeting since
+	// `time`
+	ApplauseSince(time int64) (map[int]int, error)
+}
+
+// Applause holds the state of the service.
+type Applause struct {
+	backend   Backend
+	topic     *topic.Topic
+	datastore datastore.Getter
+}
+
+// New returns an initialized state of the notify service.
+//
+// The New function is not blocking. The context is used to stop a goroutine
+// that is started by this function.
+func New(b Backend, db datastore.Getter, closed <-chan struct{}) *Applause {
```

Comment:
Do not use just one single character for a variable's name!

Reply:
If you like, you can take the responsibility for this service. If you want me to do the work, I do it in the go style.

Merge, if you like. I will not change this.